content/en/docs/tasks/observability/logs/access-log/test.sh

```diff
@@ -46,7 +46,8 @@ _verify_elided snip_test_the_access_log_1 "$snip_test_the_access_log_1_out"
 
 # Check the logs
 _verify_contains snip_test_the_access_log_2 "outbound|8000||httpbin.default.svc.cluster.local"
-_verify_contains snip_test_the_access_log_3 "inbound|8000|http|httpbin.default.svc.cluster.local"
+_verify_contains snip_test_the_access_log_3 "inbound|8000||"
+_verify_contains snip_test_the_access_log_3 ".httpbin.default.svc.cluster.local"
```

Comment:
Can the check be more specific?
```suggestion
_verify_contains snip_test_the_access_log_3 "outbound_.8000_._.httpbin.default.svc.cluster.local"
```

Reply:
What I did was separate the prior check into two checks for the pieces that surround the new data, so we are checking for the same things as before.  If we had a regex check we could have the old check and say to ignore all the characters in the middle of the string.

Reply:
If you want to check two things, you should use _verify_lines instead of two _verify_contains calls (you don't want to call the snip twice).
```
_verify_lines snip_test_the_access_log_3 "
+ inbound|8000||
+ .httpbin.default.svc.cluster.local"
```
That said, I think my suggested check (above) is a better and simpler check that the request was logged.

Reply:
I updated to use the single check you mentioned above.